app/src/main/java/org/wikipedia/analytics/ABTestDescriptionEditChecksFunnel.java

```diff
@@ -0,0 +1,20 @@
+package org.wikipedia.analytics;
+
+public final class ABTestDescriptionEditChecksFunnel extends ABTestFunnel {
+    public ABTestDescriptionEditChecksFunnel() {
+        super("descriptionEditChecks", ABTestFunnel.GROUP_SIZE_2);
+    }
+
+    public boolean shouldSeeChecks() {
+        boolean enrolled = isEnrolled();
+        boolean showChecks = getABTestGroup() == ABTestFunnel.GROUP_1;
```

Comment:
Would it be possible to add `ReleaseUtil.isPreBetaRelease()` for testing purposes?

Reply:
Yep